packages/app-mobile/root.tsx

```diff
@@ -822,6 +826,60 @@ class AppComponent extends React.Component {
 		}
 	}
 
+	private async openCallbackUrl(url: string) {
```

Comment:
Could you move this function to a separate file please?

Reply:
Done